x-pack/plugins/cases/public/common/mock/test_providers.tsx

```diff
@@ -129,7 +129,7 @@ export interface AppMockRenderer {
   render: UiRender;
   coreStart: StartServices;
   queryClient: QueryClient;
-  AppWrapper: React.FC<{ children: React.ReactElement }>;
+  AppWrapper: React.FC<{ children: React.ReactNode }>;
```

Comment:
I got an ts error here https://github.com/elastic/kibana/pull/171176/files#diff-5ce927c2e523c85bb1853fd933c3469c1881f69eef57ff75be97e622b18621a0R54. Looks like changing this does not break anything

Reply:
I tried but it returns another ts error 
```
Type 'FC<{ children: ReactNode; }>' is not assignable to type 'WrapperComponent<{ systemFilterConfig: FilterConfig[]; onFilterOptionChange: FilterChangeHandler; }> | undefined'.
```
Any idea what it means?

Reply:
I tried but it returns another ts error 
```
Type 'FC<{ children: ReactNode; }>' is not assignable to type 'WrapperComponent<{ systemFilterConfig: FilterConfig[]; onFilterOptionChange: FilterChangeHandler; }> | undefined'.
```
Any idea what it means?